adapter/internal/operator/apis/dp/v1alpha1/api_types.go

```diff
@@ -55,6 +55,11 @@ type APISpec struct {
 	// +optional
 	DefinitionFileRef string `json:"definitionFileRef"`
 
+	// DefinitionFileEndpoint contains the path to expose the API definition.
+	//
+	// +optional
+	DefinitionFileEndpoint string `json:"definitionFileEndpoint"`
```

Comment:
wbt renaming to below?
```suggestion
	DefinitionPath string `json:"definitionPath"`
```

Reply:
add default value as /apidefinition here. there's +kubebuilder:validation:default=<> sort of support in here. check on other places, they have these validations

Reply:
ack